neptune-python-utils/neptune_python_utils/batch_utils.py

```diff
@@ -107,7 +118,7 @@ def add_edge(t, row, **kwargs):
     for key, value in row.items():
         mapping = mappings.mapping_for(key)
         if not mapping.is_token():
-            t = t.property(mapping.name, mapping.convert(value))
+            t = add_property(t, mapping, value)
```

Comment:
In TinkerPop, edges cannot have properties with more than a single value.  Not sure how we want to handle that.  Maybe throw an exception?